src/plotting/particles.jl

```diff
@@ -1,12 +1,12 @@
-export plot_cyclotron, plot_particle
+export plot, plot
 
 """
-    plot_cyclotron(p::MagneticParticle; use_cell=true, kwargs...)
+    plot(p::MagneticParticle; use_cell=true, kwargs...)
 Plot the circle traced by the free particle motion. Optionally use `p.current_cell` for
 the particle's position. The user provided `kwargs...`
 are passed onto `PyPlot.plt[:Circle]`.
 """
-function plot_cyclotron(p::MagneticParticle; use_cell=true, kwargs...)
+function plot(p::MagneticParticle; use_cell=true, kwargs...)
```

Comment:
It is now ambiguous when you call `plot(p::MagneticParticle)` of what should happen. Should it plot the cyclotron method, or should it plot the particle method (arrow and point) ?

Reply:
Reverting this one for now.